components/shared/Table.js

```diff
@@ -1,3 +1,3 @@
-import BootstrapTable from 'react-bootstrap-table-next';
+import Table from 'react-bootstrap/Table';
```

Comment:
@santiagofn this breaks the storybook example at `stories/index.stories.js`. Can you please fix it?

Reply:
My bad 2, didn't notice this! Fixed now.